engine/src/physics/solver/penetration_constraint/physics_constants_pc.hpp

```diff
@@ -0,0 +1,36 @@
+/// @file
+/// @brief Resource @ref cubos::engine::MagiConfig.
+/// @ingroup physics-plugin
+
+#pragma once
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Resource which allows configuration over magic numbers in the solver plugin.
```

Comment:
We should avoid using magic numbers to describe something haha. Could be like this
```suggestion
    /// @brief Resource which allows configuration over constants in the penetration constraint plugin.
```